lld/ELF/LinkerScript.h

```diff
@@ -256,6 +256,11 @@ struct InsertCommand {
   StringRef where;
 };
 
+struct CrossRefList {
+  SmallVector<StringRef, 2> refs;
+  bool firstOnly;
```

Comment:
I found firstOnly a bit too tied the implementation and needed a bit of referring to the documentation. I guess it is firstOnly as when we are 

Perhaps worth adding a comment
```
See documentation for NOCROSSREFS and NOCROSSREFS_TO. When firstOnly is false refs are output section names that must not have any cross references between them. When firstOnly is true. The first ref is the tosection and the rest are fromsections.